css/styles.css

```diff
@@ -0,0 +1,19 @@
+:root {
+  --black: #000;
+  --wine: #730d15;
+  --red: #f00;
+  --gray: gray;
+  --gray2: #262626;
+  --white: #fff;
+}
+
+#navbar {
+  /* position: absolute */
```

Comment:
You should not leave unused lines of code as comments unless there is a reason why that commented code is there. If there is no valid reason delete those comments that make your code not clean. 🧹